chart/templates/kratos-config.yaml

```diff
@@ -4,7 +4,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: kratos-config
+  name: mission-control-kratos-config
```

Comment:
Is there a reason for this ?

Reply:
Yes - we cannot overwrite the configmap created by the Kratos chart (see https://github.com/flanksource/mission-control-chart/actions/runs/4859934069/jobs/8663167611#step:7:39 for example). So this creates our config in a new configmap and maps that to a second config file that takes preference over the one provided by the Kratos chart